test/lit/passes/multi-memory-lowering.wast

```diff
@@ -0,0 +1,65 @@
+;; NOTE: Assertions have been generated by update_lit_checks.py and should not be edited.
+;; RUN: wasm-opt %s --enable-multi-memories --multi-memory-lowering --enable-bulk-memory --enable-extended-const -S -o - | filecheck %s
+
+(module
+  (memory $memory1 1)
+  (memory $memory2 2)
+  (data (memory $memory1) (i32.const 0) "a")
+  (data (memory $memory2) (i32.const 1) "123")
+)
+
+;; CHECK: (module
+ ;; CHECK-NEXT:(type $none_=>_i32 (func (result i32)))
```

Comment:
Did update_lit_checks.py emit these unaligned checks? This seems like a bug in the script if so. It looks like CHECK prefix and the whitespace that is supposed to come after it have been swapped.

Reply:
Talked offline